_sass/minima/layout/_docs.scss

```
@@ -19,12 +19,15 @@
 .toc .h2 {
     font-size: 17px;
     padding: 5px 0px;
+    font-weight: 200;
     a {
         color: #ffffff;
         text-decoration: none;
         list-style-type: none;
+        line-height: 28px;
         &:hover {
-            color: #AFDEED;
+          color: #f29111;
+          font-weight: 400;
```

```suggestion
          text-decoration: underline;
```
This is to prevent the flashing in the left nav

---

I removed font-weight changing. As for underlining, I prefer not to add it.